tests/components/homekit/test_util.py

```diff
@@ -199,27 +204,36 @@ async def test_show_setup_msg(hass):
     """Test show setup message as persistence notification."""
     pincode = b"123-45-678"
 
-    call_create_notification = async_mock_service(hass, DOMAIN, "create")
+    entry = await async_init_integration(hass)
+    assert entry
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5453535)